boot/zephyr/Kconfig

```
@@ -1283,10 +1283,17 @@ config USB_DEVICE_PRODUCT
 config MCUBOOT_BOOTUTIL_LIB_OWN_LOG
 	bool
 
+config MCUBOOT_USE_CHECK_LOAD_ADDR
+	bool "use check of load address"
```

```suggestion
	bool "Use check of load address"
```
if we're adding to all new images, do we want to `default y` this? Or I guess not right away so other things e.g. qspi xip can be updated